convert2rhel/backup.py

```diff
@@ -105,7 +105,14 @@ def _install_local_rpms(self, pkgs_to_install, replace=False, critical=True):
             pkgs_as_str = " ".join(pkgs_to_install)
             loggerinst.debug(output.strip())
             if critical:
-                loggerinst.critical("Error: Couldn't install %s packages." % pkgs_as_str)
+                loggerinst.critical_no_exit("Error: Couldn't install %s packages." % pkgs_as_str)
+                raise exceptions.CriticalError(
+                    id_="COULDNT_INSTALL_PACKAGES",
+                    title="Couldn't install packages.",
+                    description="XXX DESCRIPTION.",
+                    diagnosis="Couldn't install %s packages. Command: %s Output: %s Status: %d"
+                    % (pkgs_as_str, cmd, output, ret_code),
+                )
```

Comment:
I would probably change to Failed to install packages or something, we can ask Miriam in the spreadsheet though! Same with the other messages in the PR

Reply:
Yeah, I got that feeling while editing the spreadsheet. Will fix when I fuss with the description.